packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -179,12 +179,23 @@ function getOffset(node) {
   };
 }
 
-function getMousePosition(event) {
-  if (event.changedTouches && event.changedTouches[0]) {
-    return {
-      x: event.changedTouches[0].pageX,
-      y: event.changedTouches[0].pageY,
-    };
+function getMousePosition(event, touchId) {
+  if (event.changedTouches) {
+    let touchIndex = 0;
+    for (let i = 0; i < event.changedTouches.length; i += 1) {
+      const touch = event.changedTouches.item(i);
+      if (touch.identifier === touchId) {
+        touchIndex = i;
+        break;
+      }
+    }
+
+    if (event.changedTouches[touchIndex]) {
+      return {
+        x: event.changedTouches[touchIndex].pageX,
```

Comment:
Sometimes you're using `TouchList.prototype.item` sometimes direct index access. For better readability I would suggest you include a `TouchList.prototype.identifiedTouch` which looks like this is what you're doing in the `for` loop.